go.mod

```diff
@@ -156,6 +155,7 @@ require (
 	github.com/subosito/gotenv v1.2.0 // indirect
 	github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca // indirect
 	github.com/tecbot/gorocksdb v0.0.0-20191217155057-f0fad39f321c // indirect
+	github.com/tendermint/tm-db v0.6.6 // indirect
```

Comment:
just noting that tm-db is still a dep in case that was causing issues (I remember some slack discussions around this with @jbowen93).

Reply:
This is fine.